sql/PageApprovals.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE /*_*/approval_log (
+    al_id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
For me, AUTOINCREMENT doesn't work. I think for MySQL the correct syntax is AUTO_INCREMENT. And AUTOINCREMENT perhaps is for SQLite.

![image](https://github.com/ProfessionalWiki/PageApprovals/assets/15813104/623ada88-11b4-4b5e-be44-4c2b1ce380f0)

Reply:
Of course, it will not throw any error if the Tables are already there.

Reply:
I originally had `AUTO_INCREMENT`, but then it failed on GitHub.

Reply:
I'm not quickly finding a way to make it work with both. I thought MediaWIki took care of that.

I am out till this evening. Feel free to try fixing this.